mad-dog/src/tests/test_userReplicaSetManager.js

```diff
@@ -34,14 +34,16 @@ const verifyUserReplicaSetStatus = async (
     let usrQueryInfo = await getUser(libs, userId)
 
     // Validate the latest updater indexed into discovery node matches expected value
-    let replicaSetUpdateSigner = replicaSetUpdaterAddress
+    let queriedReplicaSetUpdateSigner = usrQueryInfo.replica_set_update_signer.toLowerCase()
+
+    let validReplicaSetUpdateSigner = [replicaSetUpdaterAddress]
     if (!replicaSetUpdateSigner) {
```

Comment:
is this variable defined anywhere? did you mean `replicaSetUpdaterAddress`?

Reply:
/ did you mean to remove it

Reply:
Yeah, good callout, changing